crates/viewer/re_data_ui/src/entity_db.rs

```diff
@@ -227,5 +213,32 @@ impl crate::DataUi for EntityDb {
                 }
             }
         }
+
+        ui.section_collapsing_header("Properties").show(ui, |ui| {
```

Comment:
This has a weird interaction with the top-level `Data` collapsing header:

Collapse properties:
<img width="418" alt="image" src="https://github.com/user-attachments/assets/3f4bbb52-6a2d-4d48-9c15-294e7ee0a48b" />

Collapse Data:

<img width="407" alt="image" src="https://github.com/user-attachments/assets/36e61a14-13af-41fb-b092-e4cc6af224d7" />

(I'd expect "Properties" to still be visible)

My  suggestion would be to NOT add this here, but instead add it to `re_selection_panel` when the recording is selected.